Hunting Queries/SecurityAlert/WebShellFileAlertEnrich.yaml

```diff
@@ -33,3 +33,16 @@ query: |
   ) on FileName 
   | project StartTime, EndTime, AttackerIP, AttackerUserAgent, SiteName, ShellLocation
   | extend timestamp = StartTime, IPCustomEntity = AttackerIP  
+
+metadata:
+  version: 1.0.0
+  source:
+    kind: Standalone
+  author:
+    name: Microsoft 
+  support:
+    tier: Microsoft
```

Comment:
For support, would it better to have a link so that the user in portal can redirect to the support page.

Reply:
we have default values if support is not added - 
``` 
$link = $support.link ?? $(
        if ($tier -eq "Microsoft") { "https://support.microsoft.com" } else {
            if ($tier -eq "Community") {
                "https://github.com/Azure/Azure-Sentinel/issues"
            } else {
                $null
            }
        }
        );
```